examples/azurestack/kubernetes-azurestack-azure-ad.json

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Could you please run "format document" on these two?

Reply:
fixed